.github/workflows/main.yml

```
@@ -0,0 +1,23 @@
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+      - name: Test with Checkov
+        id: checkov
+        uses: bridgecrewio/checkov-action@master
+        with:
+          directory: .
+          framework: terraform 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)